### DISCUSSION TITLE
Can I specify how many frames to read?

### DISCUSSION THREAD

**Discussion by zengqingfa** [bluenviron/gortsplib#75] [Questions]
Hello, I want to save the video every half an hour, may I ask how to achieve?

**Comment by aler9** [comment]
Hello, this library doesn't support decoding H264 (only receiving it), in order to do so you can use ffmpeg inside a loop:

```/bin/sh
while true; do
    ffmpeg -re -i rtsp://... -vframes 1 -q:v 2 output.jpg
    sleep 1800
done
```
